src/woo_publications/utils/utils.py

```diff
@@ -0,0 +1,17 @@
+import re
+from typing import Union
+
+from djangorestframework_camel_case.util import (
+    camelize_re,
+    underscore_to_camel as _underscore_to_camel,
+)
+
+
+def underscore_to_camel(input_: Union[str, int]) -> str | Union[str, int]:
```

Comment:
with the new python 3.12 typing syntax, I think this should be:

```py
def underscore_to_camel[T: str | int](input_: T) -> T:
```
explanation:

1. the function takes a generic type `T`, which is either a `str` or an `int`
2. the function returns the same type as the input type (str -> str, and int -> int. str -> int or int -> str are not possible)
3. `str | int` is nicer syntax for `Union[str, int]`

`str | Union[str, int]` is also weird, since you're basically saying "it's `str` or `str` or `int`" -> the `str` union member is duplicated